### ISSUE TITLE
Selecting Top Item in Directory Causes Refresh to Become Stuck on Screen

### ISSUE THREAD

**Issue by thewizardstudio** [SimpleMobileTools/Simple-File-Manager#175]
#### Expected Behavior
Tapping and Holding on the top File or Folder in a Directory list Selects that item.

#### Actual Behavior
Tapping and Holding on the top File or Folder in a Directory list causes the Refresh icon to become stuck on screen.  This can block the User's view of interactable elements beneath it.

#### How to Reproduce
1. Launch File Manager 3.2.0 on Android 8.1.0
2. Navigate to a Directory containing Files or Folders
3. Tap and Hold on the top item in the list to Select

---
    Device: Pixel 2 XL
    OS: Android 8.1.0
    App Version: 3.2.0
---

#### Recording of the Bug
![SelectRefreshStuck.gif](https://res.cloudinary.com/hpiynhbhq/image/upload/v1517514567/eag6cyxetqvtoods8zfr.gif)

<br /><hr/><em>Posted on <a href="https://utopian.io/utopian-io/@thewizard.studio/selecting-top-item-in-directory-causes-refresh-to-become-stuck-on-screen">Utopian.io -  Rewarding Open Source Contributors</a></em><hr/>

**Comment by landry314** [comment]
yes, this is really annoying. if i am trying to get information or rename or move or copy the top item on the screen the refresh icon comes down and stays there until I swipe it up and away.
